spec/factories/tasks.rb

```diff
@@ -1,9 +1,16 @@
 FactoryBot.define do
   factory :task do
+    association :project
     title { 'Task' }
     status { rand(2) }
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
   end
+
+  trait :task_completed_yesterday do
+    status { 'done' }
```

Comment:
enumの値はSymbol形式で記載して、ただの文字列ではないことを強調しましょう！
```suggestion
    status { :done }
```

Reply:
ありがとうございます。
https://github.com/daigomiyoshi/rspec_app_exam/pull/1/commits/a4ad5daadca11f15584cfa2e5d67b2961e8f0e54 にて修正いたしました！